src/helpers/index.ts

```diff
@@ -1,5 +1,42 @@
-import {Platform} from 'react-native';
+import {Animated, Platform} from 'react-native';
 
 export const isAndroid = (): boolean => {
```

Comment:
instead of function you can use this like
`export const isAndroid = Platform.OS === 'android';
`